src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -71,6 +73,9 @@ public ObservableList<Contact> getFilteredContactList() {
         return model.getFilteredContactList();
     }
 
+    @Override
+    public ObservableList<Team> getTeamList() {return model.getCodeConnect().getTeamList();}
```

Comment:
You should add line breaks to the body of this method.
```suggestion
    public ObservableList<Team> getTeamList() {
        return model.getCodeConnect().getTeamList();
    }
```